.github/workflows/ci.yml

```diff
@@ -46,7 +46,7 @@ jobs:
       - name: Code format check
         run: cargo fmt --check -- --config "unstable_features=true,imports_granularity=Crate,group_imports=StdExternalCrate"
       - name: Clippy
-        run: cargo clippy --all --examples -- -D warnings
+        run: cargo clippy --workspace --all-targets --all-features -- --deny warnings
```

Comment:
@JEnoch I updated the clippy argument here to match the one we used in Zenoh. I added `--all-targets` and `--all-features` to avoid the possible changes in the future. Also, `--all` is replaced by `--workspace`.